test/rulesets/Base/base.jl

```diff
@@ -80,21 +80,6 @@
         end
     end
 
-    @testset "matmul *(x, y)" begin
```

Comment:
Has this been superseded by some other tests somewhere?

Reply:
There's already a matrix multiplication `rrule` test in the `dense.jl` file in `test`, so this is redundant. And the rule itself is defined in `dense.jl` in `src`.